08.Lesson_WorkingWithInvariants/InvariantsConcepts/Manager/run.sh

```diff
@@ -1 +1 @@
-certoraRun Manager.sol:Manager --verify Manager:ManagerFullSolution.spec --solc solc8.6 --send_only --msg "$1"
+certoraRun Manager.sol:Manager --verify Manager:ManagerFullSolution.spec --solc solc8.6 --send_only --msg "Manager lesson 8" --staging master --rule_sanity advanced
```

Comment:
will "--staging master" work for users? I know that "--staging" is enough, but idk if the tool will through an error for them with "--staging master"